d2bs/kolbot/libs/MuleLogger.js

```diff
@@ -27,7 +27,7 @@ var MuleLogger = {
 	LogEquipped: false, // include equipped items
 	LogMerc: false, // include items merc has equipped (if alive)
 	SaveScreenShot: false, // Save pictures in jpg format (saved in 'Images' folder)
-	IngameTime: 20, // Time to wait after leaving game
+	IngameTime: rand(180, 210), // Time to wait in game to do not get RD, or set the value rand(7230, 7290) for mule perming
```

Comment:
Change comment to `// 180, 210 to avoid RD, increase to 7230, 7290 for mule perming`